go/client/cmd_simplefs_stat.go

```diff
@@ -101,7 +113,7 @@ func (c *CmdSimpleFSStat) Run() (err error) {
 				r.Revision, keybase1.FormatTime(e.Time),
 				keybase1.DirentTypeRevMap[e.DirentType],
 				e.Size, e.Name, e.LastWriterUnverified.Username,
-				e.PrefetchStatus)
+				prefetchStatusString(e))
```

Comment:
I feel like this output has just gotten less and less human-intelligible. Do we intend for humans to read the output of this command, or just scripts?

Reply:
Yeah this output is a mess, I agree.  But we've never had a proper design for it.  I guess we just need headers or something, but that seems like the job for a future PR, if anyone cares enough to do it.